### ISSUE TITLE
Improve storybook user experience

### ISSUE THREAD

**Issue by klappradla** [bitcrowd/bitstyles#624]
I was wondering if Storybook (or some addon to it) could allow us to add more tags or metadata to stories to improve the search. It would for instance be super neat, if one could get anything related to `flexbox` by searching for `"flex"`.

**Comment by planktonic** [comment]
@klappradla there are some addons that do similar things, does any of them fit?

- https://storybook.js.org/addons/@geometricpanda/storybook-addon-badges
- https://storybook.js.org/addons/@etchteam/storybook-addon-status

To me they seem quite limited, not intended to be used for general metadata really, but maybe it’s enough, wdyt?

**Comment by klappradla** [comment]
Just from a brief look: the badges, etc. don't seem to be integrated into the search. Without that, they feel like unnecessary visual clutter to me. Maybe I just have a different mental modal of the structure of things in the library. E.g. finding the `u-items-` utility, the first things I'd look for are "flex" and "align", not "items". So I'd personally group the utilities one level further (e.g. with something like "Flexbox" here or "Box Model" elsewhere). But that may just be my personal view on the topic.

**Comment by planktonic** [comment]
Yes they don’t seem to show up in search, true 🤔 We could nest the utility classes e.g. `utilities > flex > items`, `utilities > flex > justify` (is that what you mean?) so they’d show up when searching for `flex`, but those classes are for grid layouts too. A docs file can’t appear multiple times. I don’t see a great solution right now, I’ll keep thinking about it

**Comment by klappradla** [comment]
> I don’t see a great solution right now

Yep, that's pretty much what I wanted to say with my [previous comment](https://github.com/bitcrowd/bitstyles/issues/624#issuecomment-1031415276).
